linux/files/collectd_df.conf

```diff
@@ -1,9 +1,10 @@
-LoadPlugin df
+<LoadPlugin df>
+  Globals false
+</LoadPlugin>
+
 <Plugin df>
   {%- for fs_type in plugin.get('fs_types', []) %}
   FSType {{ fs_type }}
   {%- endfor %}
-  {%- if plugin.get('ignore_selected', False) %}
-  IgnoreSelected true
-  {%- endif %}
+  IgnoreSelected {{ plugin.get('ignore_selected', true)|lower }}
```

Comment:
Minor, but wouldn't it make more sense to use

```
plugin.get('ignore_selected', True)|lower
```

or

```
plugin.get('ignore_selected', 'true')|lower
```

?

Just a suggestion…